portal/templates/patients_by_org.html

```diff
@@ -20,7 +20,7 @@ <h4 class="tnth-headline">Patient List</h4>
               </div>
               <div class="modal-body">
                   <div class="form-group">
-                    <label><em class="text-muted">{{ _("Instrument(s) to export data from:")}}</em></label>
+                    <label class="text-muted">{{ _("Instrument(s) to export data from:")}}</label>
```

Comment:
We may be able to set the [correct plural form in the source and translation](https://developer.mozilla.org/en-US/docs/gettext#Plurals), but I still need to do some research on how we would implement it.